libs/elasticsearch/pyproject.toml

```diff
@@ -25,8 +25,9 @@ pytest-mock = "^3.10.0"
 syrupy = "^4.0.2"
 pytest-watcher = "^0.3.4"
 pytest-asyncio = "^0.21.1"
-langchain = "^0.1.13"
-langchain-text-splitters = ">=0.0.1,<0.1"
+langchain = {git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/langchain"}
+langchain-community = {git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/community"}
+langchain-text-splitters = {git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/text-splitters"}
```

Comment:
These were changed by mistake. And I believe we do not need `langchain-community` in here.

Reply:
yep my mistake, removed.